### PR TITLE
Sidebar update

### LINKED ISSUE #125: Side Bar Navigation
**Is your feature request related to a problem? Please describe.**
The sidebar navigation needs to be updated

**Describe the solution you'd like**
Update styling and remove not needed props

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Created a Sidebar nav component to house the buttons to switch between sidebars

## Purpose
Since the same logic was being used in two places, it seemed like a good time to create a new component

## Approach
Created a new component that housed the nav buttons. Passed the new component the necessary props. 

Closes #125

### CODE REVIEW

**Review by gonzalespaulb — APPROVED** [review]
The two sidebar states look more cohesive now that they share a similar component

**Review by BlackCubes — APPROVED** [review]
It's looking more and more organized. Noice!

File: pokeapp/src/components/SidebarNav.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          setbadgeBtnActive1(true);
+          setbadgeBtnActive2(false);
+          setIsBadgeSideBar(true);
+        }}
+      >
+        Badges
+      </div>
+      <div
+        className={
+          badgeBtnActive2 ? "badge-btn-right active" : "badge-btn-right"
```
Comment (Lap343) [inline pokeapp/src/components/SidebarNav.js:19]: nice so what I am getting is they come in and out depending on the state the user chooses

**Review by Lap343 — APPROVED** [review]
code looks good. Good job

**Review by JorgeLVilla — APPROVED** [review]
Looking good! I really want to see these badges on the app!

**Pull request merged**